packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -259,15 +261,14 @@ type ScrollAreaInternalProps = {
 };
 
 type ScrollAreaNativeProps = ScrollAreaInternalProps &
-  Omit<ScrollAreaOwnProps, 'unstable_forceNative'> & {
+  Omit<ScrollAreaOwnProps, 'unstable_forceNative' | 'selector'> & {
```

Comment:
technically should `as` be omitted here too then?

Reply:
Oooh good catch. I shouldn't have removed `selector` here. They should be able to pass `selector` and `as`.